NMS-14301: Configure port for juniper devices (#4762)
OpenNMS/opennms: docs/modules/operation/pages/device-config-backup/configuration.adoc

--
====

== Configure TFTP IPAddress

The device config backup feature allows the retrieval of network device configurations by using the TFTP

> protocol. To achieve this, a TFTP server needs to be started on the OpenNMS instance or on a minion. Normally

Reviewer: ```suggestion

To achieve this, a TFTP server needs to be started on the OpenNMS instance or on a Minion. 
```
Separate lines

* `eventReason` - If the backup failed, contains a short description of the issue.

> == Backup device configuration on demand

Reviewer: ```suggestion
== Back up device configuration on demand
```

> You can manually backup devices from OpenNMS UI under Info>Device Configs.

Reviewer: ```suggestion
You can manually back up devices from the OpenNMS UI under *Info>Device Configs*.
```

> You can also use the Rest API to trigger device configuration retrieval outside of a monitor schedule.

Reviewer: ```suggestion
You can also use the REST API to trigger device configuration retrieval outside of a monitor schedule.
```
